.github/workflows/golden_tests.yml

```
@@ -0,0 +1,101 @@
+      - name: Upload golden test report
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: golden-test-report
+          path: golden_test_report.md
+          retention-days: 14
+
+      # Add a comment to the PR with the results
+      - name: Comment on PR
+        if: github.event_name == 'pull_request' && steps.golden-test-run.outputs.has_differences == 'true'
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)